### PR TITLE
Make conversion facultative

### PR DESCRIPTION
**Pull request by ElsLommelen**
This PR solves issue #11:
- adapt encoding for varchar but not for nvarchar (to show a maximum of symbols correctly)
- logical argument `autoconvert_utf8` makes encoding facultative for users
- remove `convertdf_enc()` (moved back to package [inborutils](https://github.com/inbo/inborutils/pull/86))

I moreover added an [issue](https://github.com/r-dbi/odbc/issues/371) in the odbc package on the encoding problem in windows

### CODE REVIEW

File: R/connect_inbo_dbase.R [hunk 1]
```diff
@@ -17,14 +19,17 @@
...
 #' @author Stijn Van Hoey \email{stijnvanhoey@@gmail.com}
 #' @examples
 #' \dontrun{
 #' connection <- connect_inbo_dbase("D0021_00_userFlora")
 #' connection <- connect_inbo_dbase("W0003_00_Lims")
 #' }
-connect_inbo_dbase <- function(database_name) {
+connect_inbo_dbase <- function(database_name, autoconvert_utf8 = TRUE) {
+
+    assert_that(is.logical(autoconvert_utf8))
```
Comment (florisvdh) [inline R/connect_inbo_dbase.R]:
```suggestion
    assert_that(is.flag(autoconvert_utf8), noNA(autoconvert_utf8))
```

File: R/connect_inbo_dbase.R [hunk 2]
```diff
@@ -17,14 +19,17 @@
 #' @importFrom DBI dbGetQuery dbConnect dbListTables
 #' @importFrom odbc odbc odbcListDrivers
 #' @importFrom utils tail
+#' @importFrom assertthat assert_that
```
Comment (florisvdh) [inline R/connect_inbo_dbase.R]:
```suggestion
#' @importFrom assertthat
#' assert_that
#' is.flag
#' noNA
```

File: R/odbc_dbFetch.R [hunk 3]
```diff
@@ -1,11 +1,13 @@
 #' Fetch query result from database
 #'
-#' This method is an adaptation from the eponymous function in the `odbc` package and is an implementation of the method `dbFetch` defined in the `DBI` package.  Additional to the `odbc` package, it replaces an cryptic error message by an informative error message.  In Windows OS, it moreover assures that characters are interpreted correctly (while their presentation depends on the settings of the locale).
+#' This method is an adaptation from the eponymous function in the `odbc` package and is an implementation of the method `dbFetch` defined in the `DBI` package.  Additional to the `odbc` package, it replaces an cryptic error message by an informative error message.  In Windows OS and if chosen encoding = "utf-8" in connect_inbo_dbase(), it moreover assures that characters are interpreted correctly (while their presentation depends on the settings of the locale).
```
Comment (florisvdh) [inline R/odbc_dbFetch.R]:
```suggestion
#' This method is an adaptation from the eponymous function in the `odbc` package and is an implementation of the method `dbFetch` defined in the `DBI` package.  Additional to the `odbc` package, it replaces an cryptic error message by an informative error message.  In Windows OS and if chosen `autoconvert_utf8 = TRUE` in connect_inbo_dbase(), it moreover assures that characters are interpreted correctly (while their presentation depends on the settings of the locale).
```

Also Rd file is to be updated if you commit this.

**Review by florisvdh — COMMENTED** [review]
:+1:  With a few minor suggestions.

**Comment by ElsLommelen** [comment]
Note: I finally simplified the code by using param encoding in dbConnect(), which gives the same result.  Only less common symbols stay a problem when used in SQL Server data type varchar

**Review by hansvancalster — APPROVED** [review]
Looks good. Great to see there is a simple solution.

**Comment by florisvdh** [comment]
Great move @ElsLommelen , never expected that it could be solved in such an easy way :smile: .

**Pull request merged**